src/lib/core/CHIPError.h

```diff
@@ -2435,6 +2435,14 @@ using CHIP_ERROR = ::chip::ChipError;
  */
 #define CHIP_ERROR_HANDLER_NOT_SET                            CHIP_CORE_ERROR(0xe1)
 
+/**
+ * @def CHIP_ERROR_INPROGRESS
+ *
+ * @brief
+ *   The operation is still in progress
+ */
+#define CHIP_ERROR_INPROGRESS                                 CHIP_CORE_ERROR(0xe2)
```

Comment:
There's an existing gap at 0xa4.  Could you please use that value instead of adding this at the end?

Reply:
Also, docs/ERROR_CODES.md needs to be updated, right?

Reply:
Thank you @bzbarsky-apple 
I have updated its value to 0xa4 and updated the table in ERROR_CODES.md.

- Please take a look.

Reply:
@bzbarsky-apple @Kxuan , does CHIP_ERROR_INPROGRESS is in the specs ?
i am not against it but maybe the specs need to be updated following this behavior

Reply:
@jmeg-sfy CHIP_ERROR values do not generally have anything to do with specs...